packages/combobox/test/combobox-a11y.test.ts

```diff
@@ -0,0 +1,309 @@
+            expect(a11yNode, '`name` is null on WebKit').to.not.be.null;
+        } else {
+            a11yNode = findAccessibilityNode<NamedNode>(
+                snapshot,
+                (node) =>
+                    node.name === name &&
+                    node.value === 'Bde Thing 2' &&
+                    node.role === 'combobox'
+            );
+            expect(
+                a11yNode,
+                '`name` is the the selected item text plus the label text'
+            ).to.not.be.null;
+        }
+    });
+    it('manages its "name" value in the accessibility tree', async () => {
+        const el = await comboboxFixture();
+
+        const name = 'Combobox';
+        const webkitName = 'Combobox Bde Thing 2';
+        type NamedNode = { name: string; role: string; value?: string };
+
+        await elementUpdated(el);
+
+        let snapshot = (await a11ySnapshot({})) as unknown as NamedNode & {
+            children: NamedNode[];
+        };
+
+        let a11yNode = findAccessibilityNode<NamedNode>(
+            snapshot,
+            (node) =>
+                node.name === name && !node.value && node.role === 'combobox'
+        );
+        // by default, is there a combobox that has `name` as the label?
+        expect(a11yNode, '`name` is the label text').to.not.be.null;
+
+        el.value = 'Bde Thing 2';
+        await elementUpdated(el);
+
+        snapshot = (await a11ySnapshot({})) as unknown as NamedNode & {
+            children: NamedNode[];
+        };
+        // gating it this way is the only way I could get ALL the tests to pass on all browsers
+        if (isWebKit) {
+            a11yNode = findAccessibilityNode<NamedNode>(
+                snapshot,
+                (node) =>
+                    node.name === webkitName &&
+                    node.value === 'Bde Thing 2' &&
+                    node.role === 'combobox'
+            );
+            expect(a11yNode, '`name` is null on WebKit').to.not.be.null;
+        } else {
+            a11yNode = findAccessibilityNode<NamedNode>(
+                snapshot,
+                (node) =>
+                    node.name === name &&
+                    node.value === 'Bde Thing 2' &&
+                    node.role === 'combobox'
+            );
+            expect(
+                a11yNode,
+                '`name` is the the selected item text plus the label text'
+            ).to.not.be.null;
+        }
+    });
+    it('renders open', async () => {
+        const el = await comboboxFixture();
+
+        const opened = oneEvent(el, 'sp-opened');
+        el.open = true;
+        await opened;
+
+        await elementUpdated(el);
+        await expect(el).to.be.accessible();
+    });
+    it('renders with an active descendent', async () => {
+        const el = await comboboxFixture();
+
+        const opened = oneEvent(el, 'sp-opened');
+        el.open = true;
+        await opened;
+
+        el.activeDescendant = el.availableOptions[0];
```

Comment:
Does this practically effect the UI? Or do we need to activate a descendant by interacting with the Combo Box first?

Reply:
It does not! I will change this test to be more practical.

Reply:
Actually, what's the goal here? 

The test tagline is "renders with an active descendent", but then the actual test criteria is `await expect(el).to.be.accessible();`. 

If we're going to affect the UI, would the next test, "manages `aria-activedescendant`", be sufficient if we added the `expect(el).to.be.accessible();` to that test? Or is the goal actually to render something with an active descendant by default? I can't imagine that scenario actually being a use case, though...

Reply:
That seems reasonable to me. It is pretty similar to the following test, yes.